android/app/build.gradle

```diff
@@ -210,7 +216,9 @@ dependencies {
     } else {
         implementation jscFlavor
     }
-    androidTestImplementation('com.wix:detox:+')  // detox: e2e testing
+    androidTestImplementation('com.wix:detox:+')
+    implementation "androidx.core:core-ktx:+"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version" // detox: e2e testing
```

Comment:
the comment here is for the detox line

Reply:
Odd, I think that got moved by enabling Kotlin automatically. I’ll correct.